src/main/kotlin/uk/gov/dluhc/registercheckerapi/config/EmailContentConfiguration.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.dluhc.registercheckerapi.config
+
+import com.amazonaws.util.IOUtils
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+import org.springframework.core.io.ClassPathResource
+
+/**
+ * Class to bind configuration properties for the email content
+ */
+@ConfigurationProperties(prefix = "email.pending-register-checks-content")
+@ConstructorBinding
+class EmailContentConfiguration(

```

Comment:
Could maybe rename this to make it clear it is content specifically for the monitoring pending register checks email, though appreciate there aren't any other emails to confuse it with.